homeworks/src/homework6/ArrayDataException.java

```diff
@@ -0,0 +1,15 @@
+package homework6;
+
+public class ArrayDataException extends Exception {
+
+    private static String errorMessage;
+
+    public ArrayDataException(String errorMessage) {
+        ArrayDataException.errorMessage = errorMessage;
```

Comment:
Exception already contains constructor with message, you may reuse it by super(String someValue);